.github/build.sh

```diff
@@ -4,6 +4,7 @@ SCRIPT_RELATIVE_DIR=$(dirname "${BASH_SOURCE[0]}")
 pushd "$SCRIPT_RELATIVE_DIR"/.. || exit 1
   # use local .m2 repo to speed up build
   rsync -crlDv ~/.m2 ./.m2
-  docker build -t jenkins-snyk -f .github/Dockerfile .
+  # --platform linux/amd64 is required for M1 Macs
+  docker build --platform linux/amd64 -t jenkins-snyk -f .github/Dockerfile .
```

Comment:
Why would you need to build it for amd64 on an arm64? Isn't arm64 nowadays supported by openjdk & docker?

Reply:
Right, there's [still](https://docs.docker.com/desktop/troubleshoot/known-issues/) some issues however. In this case, the scan wouldn't run unless I specified the platform.